lightning_examples/finetuning-scheduler/.meta.yml

```diff
@@ -0,0 +1,20 @@
+title: Finetuning Scheduler
+author: "[Dan Dale](https://github.com/speediedan)"
+created: 2021-11-29
+updated: 2021-11-29
+license: CC BY-SA
+build: 1
+tags:
+  - finetuning
+description: |
+  This notebook introduces the [FinetuningScheduler](https://finetuning-scheduler.readthedocs.io/en/latest/api/finetuning_scheduler.fts.html#finetuning_scheduler.fts.FinetuningScheduler)
+  callback and demonstrates the use of it to finetune a small foundational model on the
+  [RTE](https://huggingface.co/datasets/viewer/?dataset=super_glue&config=rte) task of
+  [SuperGLUE](https://super.gluebenchmark.com/) with iterative early-stopping defined according to a user-specified
+  schedule. It uses Hugging Face's ``datasets`` and ``transformers`` libraries to retrieve the relevant benchmark data
+  and foundational model weights. The required dependencies are installed via the finetuning-scheduler ``[examples]`` extra.
+requirements:
+  - git+https://github.com/speediedan/finetuning-scheduler.git@ce4f44acdbb741317370c23ed9c70b131d36540e#egg=finetuning-scheduler[examples]
+  - git+https://github.com/huggingface/transformers.git@2f4cea0899b0f2c2988f3da30dc89deaf6af7fcb#egg=transformers
```

Comment:
@Borda this transformers pre-release requirement is just required for the next couple days. I've already talked to the Hugging Face team and they indicated that 4.18 will be released in the next few days. Also, as soon as PL 1.6 is officially released, I'll be able to push finetuning-scheduler to pypi and the only requirement will be finetuning-scheduler[examples] directly from pypi! :partying_face:

Reply:
As transformers v4.18.0 has just been released today, I've just published finetuning-scheduler to pypi and this tutorial now only depends on finetuning-scheduler[examples]!